sw-emulator/lib/periph/src/hash_sha512.rs

```diff
@@ -565,6 +654,29 @@ impl HashSha512 {
         );
     }
 
+    fn pcr_hash_op_complete(&mut self) {
```

Comment:
I wasn't able to find documentation on what order everything is hashed in. Did you find it somewhere? Or just dig through the RTL?

Assuming this is the right order, this looks good to me :)